source/_docs/timeouts.md

```diff
@@ -21,7 +21,7 @@ Rules are for the good of the group, and timeouts are no exception. We've config
   <tr>
     <td><a href="https://secure.php.net/manual/en/info.configuration.php#ini.max-execution-time">PHP max_execution_time</a></td>
     <td>120 seconds</td>
-    <td>Maximum time a script can run before being terminated by the parser. Helps prevent poorly written scripts from tying up your application container's PHP workers.
+    <td>Maximum time a script can run before being terminated by the parser. This included Drush & WP-CLI commands. Helps prevent poorly written scripts from tying up your application container's PHP workers.
```

Comment:
> This included Drush & WP-CLI commands.
I believe this is correct and the max execution time does include Drush & WP-CLI commands. This is just a default, as in the case of cli commands (drush/wp-cli) it can easily be overwritten via `ini_set` to accommodate longer running jobs which are expected from time to time. If we're going to mention drush/wp-cli I think it would be good to say that the defaults can be overwritten if needed.